### ISSUE TITLE
Extension setupFile.mjs fails to load when other setupFiles provided in a multi-project setting

### ISSUE THREAD

**Issue by roryschadler** [vitest-dev/vscode#347]
### Describe the bug

When I run `yarn test` in this reproduction repo, the tests run as expected. When I use this extension to run the tests, the tests under `typescript-project/server` run fine, but those under `typescript-project/client` fail with the error below.

I get the same error if I remove the middleman `typescript-project` directory, and work in a plain two-project monorepo. I wonder if the issue is related to [this comment](https://vitejs.dev/config/server-options.html#server-fs-allow:~:text=When%20server.fs.allow%20is%20specified%2C%20the%20auto%20workspace%20root%20detection%20will%20be%20disabled.%20To%20extend%20the%20original%20behavior%2C%20a%20utility%20searchForWorkspaceRoot%20is%20exposed%3A) under `vite`'s `server.fs.allow` section: 

> When server.fs.allow is specified, the auto workspace root detection will be disabled. To extend the original behavior, a utility searchForWorkspaceRoot is exposed...

### Reproduction

https://github.com/roryschadler/vitest-non-root-monorepo

### Output

```shell
# Vitest Output Channel

[INFO 12:29:06 PM] [v0.6.1] Vitest extension is activated because Vitest is installed or there is a Vite/Vitest config file in the workspace.
[INFO 12:29:06 PM] [API] Running Vitest: v1.5.0 (/Users/roryschadler/testing/vitest-non-root-monorepo/typescript-project/vitest.workspace.ts)
[INFO 12:29:07 PM] [API] Vitest process 6638 created
[INFO 12:29:10 PM] Running 1 file(s): /Users/roryschadler/testing/vitest-non-root-monorepo/typescript-project/
[INFO 12:38:41 PM] Running 1 file(s): /Users/roryschadler/testing/vitest-non-root-monorepo/typescript-project/
[INFO 12:39:00 PM] Running 1 file(s): /Users/roryschadler/testing/vitest-non-root-monorepo/typescript-project/

---

# Error from test results

Error: Failed to load url /Users/roryschadler/.vscode/extensions/vitest.explorer-0.6.1/dist/setupFile.mjs (resolved id: /Users/roryschadler/.vscode/extensions/vitest.explorer-0.6.1/dist/setupFile.mjs). Does the file exist?
    at loadAndTransform (file:///Users/roryschadler/testing/vitest-non-root-monorepo/typescript-project/node_modules/vite/dist/node/chunks/dep-jvB8WLp9.js:53736:21)
    at async ViteNodeServer._transformRequest (file:///Users/roryschadler/testing/vitest-non-root-monorepo/typescript-project/node_modules/vite-node/dist/server.mjs:409:16)
    at async ViteNodeServer._fetchModule (file:///Users/roryschadler/testing/vitest-non-root-monorepo/typescript-project/node_modules/vite-node/dist/server.mjs:379:17)
    at async MessagePort.<anonymous> (file:///Users/roryschadler/testing/vitest-non-root-monorepo/typescript-project/node_modules/vitest/dist/vendor/index.8bPxjt7g.js:65:20)
```

### Version

v0.6.1

### Validations

- [X] Check that you are using the latest version of the extension
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vscode/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vscode/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by NickMeske4** [comment]
I just installed the latest update and just started to run into this issue as well.

**Comment by roryschadler** [comment]
Thank you for the speedy fix again @sheremet-va! `v0.6.2` resolves the issue.
